meteor/client/ui/Settings/StudioSettings.tsx

```diff
@@ -683,9 +434,7 @@ const StudioMappings = withTranslation()(
 										<span>
 											{t('Port: {{port}}, Channel: {{channel}}', { port: mapping.portId, channel: mapping.channelId })}
 										</span>
-									)) || (
-										<span>{t('Unknown device type: {{device}}', { device: TSR.DeviceType[mapping.device] })} </span>
-									)}
+									)) || <span>-</span>}
```

Comment:
Shouldn't the summaries come from the translations as well, given that the configuration parameters for the devices are now completely abstracted away to the device to be configured within the manifest? In that case it doesn't seem to make sense to keep this tight binding here, methinks? This could simply use i18next's interpolation even, with "unescaping" turned off.

Reply:
I've done a low effort implementation of summaries composed from the manifest, translation is redundant as the manifest doesn't carry any localization.

Reply:
I think that's a reasonably good, functional effort. Anything above is sugar that probably needs some more thought anyway.